lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -370,58 +336,72 @@ public function get_item_schema() {
 	 * @return array Collection parameters.
 	 */
 	public function get_collection_params() {
-		$params = parent::get_collection_params();
+		$query_params = parent::get_collection_params();
 
-		return array(
-			'page'     => $params['page'],
-			'per_page' => $params['per_page'],
-			'search'   => $params['search'],
-			'slug'     => $params['slug'],
-		);
+		$query_params['context']['default'] = 'edit';
+
+		// Remove unneeded params.
+		unset( $query_params['after'] );
+		unset( $query_params['modified_after'] );
+		unset( $query_params['before'] );
+		unset( $query_params['modified_before'] );
+		unset( $query_params['search'] );
+		unset( $query_params['search_columns'] );
+		unset( $query_params['status'] );
+
+		$query_params['orderby']['default'] = 'id';
+		$query_params['orderby']['enum']    = array( 'id', 'include' );
+
+		/**
+		 * Filters collection parameters for the font family controller.
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param array $query_params JSON Schema-formatted collection parameters.
+		 */
+		return apply_filters( 'rest_wp_font_family_collection_params', $query_params );
 	}
 
 	/**
-	 * Checks if a given request has access to read font families.
+	 * Retrieves the query params for the font family collection, defaulting to the 'edit' context.
 	 *
 	 * @since 6.5.0
 	 *
-	 * @return true|WP_Error True if the request has read access, otherwise a WP_Error object.
+	 * @param array $args Optional. Additional arguments for context parameter. Default empty array.
+	 * @return array Context parameter details.
 	 */
-	public function get_font_family_permissions_check() {
-		$post_type = get_post_type_object( $this->post_type );
-
-		if ( ! current_user_can( $post_type->cap->edit_posts ) ) {
-			return new WP_Error(
-				'rest_cannot_read',
-				__( 'Sorry, you are not allowed to access font families.', 'gutenberg' ),
-				array( 'status' => rest_authorization_required_code() )
-			);
+	public function get_context_param( $args = array() ) {
+		if ( isset( $args['default'] ) ) {
+			$args['default'] = 'edit';
 		}
-
-		return true;
+		return parent::get_context_param( $args );
 	}
```

Comment:
This sets the context param default to `edit`, since we don't need a `view` context for fonts as admin only settings.